oh_staff_ui/admin.py

```diff
@@ -126,3 +128,8 @@ class CopyrightType(admin.ModelAdmin):
 @admin.register(Copyright)
 class Copyright(admin.ModelAdmin):
     list_display = ("value", "source")
+
+
+@admin.register(Format)
+class Format(admin.ModelAdmin):
+    list_display = ("value",)
```

Comment:
Please change this to `("item", "value",)`, so we can see a link to the `ProjectItem` each is associated with.  Since these are uncontrolled values, they really only have meaning in context of the item.

Reply:
Updated to reflect, item title display in admin display